geode-core/src/test/java/org/apache/geode/distributed/internal/ClusterDistributionManagerTest.java

```diff
@@ -1,43 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
```

Comment:
Why was this unit test removed?
It seems like, given all the changes to ClusterDistributionManager it would be good to have some unit test coverage added of that class.